src/test/java/org/candlepin/subscriptions/tally/MetricUsageCollectorTest.java

```diff
@@ -68,7 +68,7 @@
 class MetricUsageCollectorTest {
 
   public static final String RHEL_ENG_ID = "69";
-  public static final String RHEL_EUS_PAG_ENG_ID = "70";
+  public static final String RHEL_ELS_PAG_ENG_ID = "204";
```

Comment:
Looks like we were missing a "Y" here
`RHEL_ELS_PAG_ENG_ID` -> `RHEL_ELS_PAYG_ENG_ID`